app/components/article-content.js

```diff
@@ -388,6 +391,51 @@ export default Component.extend(
 			}
 		},
 
+		/**
+		 * Opens a parent section of passed element if it's closed
+		 *
+		 * @returns {void}
+		 */
+		openSection(element) {
+			if (element) {
+				const section = element.closest('section[id*="section"]');
+				if (section) {
+					const header = section.previousElementSibling;
+					if (header) {
```

Comment:
What if the ref is in the "zero section" (the article section that starts directly below the article title and ends before the first H2). This section is always expanded, even on mobile.

I would consider checking if tagName === 'H2' before appending the 'open-section' class. What do you think?